### ISSUE TITLE
fix: docs-site build assumes *ix-like script environments

### ISSUE THREAD

**Issue by cmumatt** [penrose/penrose#917]
Currently /packages/doc-site/package.json uses `rm` and `cp` commands in the docs-site `build` command, which assume a *ix-like script environment.  Use platform agnostic npm analogs instead.  See PR #916 916 conversation for more details.

**Comment by wodeni** [comment]
Any progress on this one? I noticed `penrose.cs.cmu.edu` is still deployed through the other repo. Would be good to build everything from this repo and auto-deploy.

**Comment by cmumatt** [comment]
Agree.  I think we are talking about the CloudFlare project switch rather than the script change here.  When I went to switch out the projects last week, CloudFlare was issuing undocumented errors from its Pages API to its web console.  It was getting close to Josh's talk, so I decided to let it ride and re-visit later since it was up and working.  The last thing I wanted was a broken CloudFlare site the week of the talk.
